src/js/angular/core/templates/core-errors.html

```diff
@@ -19,8 +19,10 @@
                         </span>
 
                         <span ng-hide="getAccessibleRepositories().length">
-                            {{'core.errors.no.accessible.warning.msg' | translate}}<span ng-if="isRestricted">{{'core.errors.writable' | translate}}
-                        </span>{{'core.errors.repositories' | translate}}<span ng-if="canManageRepositories()">{{'core.errors.create.repository.warning.msg' | translate}}</span>.
+                            {{(isRestricted
```

Comment:
This will evaluate on each digest cycle.
It is far better to use a span with an ng-if on it with `isRestricted` as a variable as it will evaluate only on variable change.

Reply:
Fixed